Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -980,28 +1421,107 @@ def related_incidents(client, incident_id):
     return response
 
 
-def add_note_command(client, incident_id, note):
+def get_scripts(client: SimpleClient, script_id: str) -> dict[str, Any]:
+    """
+    Retrieves a single script's enriched data if `script_id` is provided,
+    and retrieves the list of scripts belonging to the IBM QRadar SOAR organization if `script_id` is not provided.
+    """
+    response = client.get(f"/scripts/{script_id}")
+    demisto.debug(f"get_scripts | {type(response)=} | {response=}")
+    return response
+
+
+def fetch_incidents(client, first_fetch_time: str, fetch_closed: bool):
+    last_fetched_timestamp = demisto.getLastRun() and demisto.getLastRun().get('time')
+    demisto.info(f'fetch_incidents {last_fetched_timestamp=} | {first_fetch_time=}')
+
+    if not last_fetched_timestamp:
+        last_fetched_timestamp = to_timestamp(first_fetch_time)
+    args = {'date-created-after': last_fetched_timestamp}  # Fetch incident from the last fetched timestamp.
+    resilient_incidents = search_incidents(client, args)
+
+    demisto_incidents = []
+    if resilient_incidents:
+        demisto.info(f'fetch_incidents retrieved {len(resilient_incidents)=} | '
+                     f'with IDs: {[incident.get("id") for incident in resilient_incidents]}')
+        #  Update last_run_time to the latest incident creation time (maximum in milliseconds).
+        last_fetched_timestamp = last_incident_creation_time = max(
+            [_incident.get("create_date") for _incident in resilient_incidents]
+        )
+        demisto.debug(f'fetch_incidents {last_incident_creation_time=}')
+        for incident in resilient_incidents:
+
+            # Only fetching non-resolved incidents if `fetch_closed` is disabled.
+            if fetch_closed or (not incident.get('end_date') and incident.get('plan_status') == 'A'):  # 'A' stands for 'Active'
```

Comment:
consider using an Enum Active with value of A it adds readablity and reduces the comment.
something like:
class PlanStatus(Enum):
    ACTIVE = 'A'
    CLOSED = 'C'